action.yml

```diff
@@ -0,0 +1,30 @@
+name: Send pending review notifications to reviewer on github-discussion
+description: 'Send pending review notifications on team-discussion'
```

Comment:
Is team-discussion here the same as github-discussion above? If so, let's standardize how we refer to it.

Reply:
changed it to github-discussion